src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,10 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
+    {comments.map(comment => (
```

Comment:
It is highly recommended to provide a check for the existence of 'comments' before using map function on it to prevent possible runtime errors.